pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>assess-workflows</artifactId>
+    <groupId>org.redhat</groupId>
+    <version>0.0.1</version>
+    <name>Kiota Fleet Manager Client Example</name>
```

Comment:
pom.xml is not needed, if you use the proper JBang plugin for your IDE.

Reply:
I couldn't get it working and hacked around a quick `pom.xml`, I can remove it 👍

Reply:
https://marketplace.visualstudio.com/items?itemName=jbangdev.jbang-vscode FTW ;-)

Reply:
removed